catalyst_voices/integration_test/landing_page_test.dart

```diff
@@ -0,0 +1,14 @@
+import 'common.dart';
+
+void main() {
+  patrol('Test Landing page', (PatrolIntegrationTester $) async {
+    await createApp($);
+    await Future<void>.delayed(const Duration(seconds: 15));
+    expect(
+      $("""
+Project Catalyst is the world's largest decentralized innovation engine for solving real-world challenges."""),
```

Comment:
Can you please get familiar with https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization and try to get the translation from our localization assets? The approach of referencing widgets exactly by the text is kinda outdated already due to problems with it.

If you reference the translation with the key you're able to run the tests in different languages, etc.

Reply:
Done